src/SFA.DAS.AdminService.Web/Views/CertificateAmend/Version.cshtml

```diff
@@ -0,0 +1,105 @@
+﻿@model SFA.DAS.AdminService.Web.ViewModels.CertificateVersionViewModel
+
+@{
+    ViewBag.Title = (!ViewData.ModelState.IsValid ? "Error: " : "") + "Standard Version";
+    Layout = "_Layout";
+}
+
+@if (Model.FromApproval)
+{
+    <a class="govuk-back-link" asp-controller="CertificateAmend" asp-action="Check" asp-route-fromapproval="true" asp-route-certificateId=@Model.Id>Back</a>
+}
+else
+{
+    <a class="govuk-back-link" asp-controller="CertificateAmend" asp-action="Check" asp-route-certificateId=@Model.Id>Back</a>
+}
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <form type="post" asp-method="Option" class="apprentice-option-form" action="" method="post" novalidate="novalidate">
+
+                @Html.HiddenFor(c => c.Id)
+                @Html.HiddenFor(c => c.Standards)
+
+
+
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset" aria-describedby="options-conditional-hint">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 class="govuk-heading-xl">
+                                What version did the apprentice take?
+                            </h1>
+                        </legend>
+
+                        <span id="options-conditional-hint" class="govuk-hint">
+                            The version will appear on the certificate.
+                        </span>
+
+
+
+
+
```

Comment:
Unsure if all the whitespace here is git's doing or the file itself? can it be shrunk a bit?